Adding "use error" severity labeled "Invalid Use"

crgee opened the pull request:
https://indigobio.atlassian.net/browse/AR-1708

Commit:
AR-1708 - Two severities: Use Error and Catch Alls

Commit:
AR-1708 - Adding descriptors for severities

Commit:
AR-1708 - Only adding use error severities

Commit:
AR-1708 - Worst whiteboard order ever

grimesjm commented on lib/logatron/const.rb:
```diff
@@ -5,14 +5,16 @@ module Logatron
   CRITICAL = 'CRITICAL'
   FATAL = 'FATAL'
   DEBUG = 'DEBUG'
+  INVALID_USE = 'INVALID_USE'
```
I forgot to call .freeze on those strings can you do that real quick?

crgee replied:
....

Commit:
AR-1708 - Let it go! Let it go! Can't hold it back anymore!

Finished in 0.Finished in 0.0095 seconds (files took 0.12603 seconds to load)
13 examples, 0 failures
